packages/core/src/external/carequality/ihe-gateway-v2/outbound/xcpd/process/schema.ts

```diff
@@ -34,6 +34,14 @@ export const identifierSchema = z.object({
 });
 export type IheIdentifier = z.infer<typeof identifierSchema>;
 
+const genderCodeSchema = z.union([
+  z.literal("F"),
+  z.literal("M"),
+  z.literal("UN"),
```

Comment:
Reminder to update this in case we update `UN`

Reply:
this stays the same. These are codes in the IHE spec which are constant